cargo-dist/src/config.rs

```diff
@@ -397,6 +403,9 @@ impl DistMetadata {
         if hosting.is_some() {
             warn!("package.metadata.dist.hosting is set, but this is only accepted in workspace.metadata (value is being ignored): {}", package_manifest_path);
         }
+        if extra_artifacts.is_some() {
+            warn!("package.metadata.dist.extra_artifacts is set, but this is only accepted in workspace.metadata (value is being ignored): {}", package_manifest_path);
+        }
```

Comment:
oh interesting, i would have assumed this was package-specific (*vaguely recalls some discussion we had about this but forgets the conclusion*)